tests/transforms/test_sign_expand.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+import pennylane as qml
+import pennylane.tape
+from pennylane import numpy as pnp
+
+"""Defines the device used for all tests"""
+
+dev = qml.device("default.qubit", wires=[0, 1, 2, 3, "Hadamard", "Target"])
+
+"""Defines circuits to be used in queueing/output tests"""
+
+with pennylane.tape.QuantumTape() as tape1:
```

Comment:
In PL we tend to use `#` for comments. `"""` is used for docstrings.
```suggestion
# Defines the device used for all tests
dev = qml.device("default.qubit", wires=[0, 1, 2, 3, "Hadamard", "Target"])

# Defines circuits to be used in queueing/output tests
with pennylane.tape.QuantumTape() as tape1:
```

Reply:
Changed